PZ/WebClient/Connector.py

```diff
@@ -0,0 +1,140 @@
+        # resource.measurements = self.request['measurements']
+        # return resource
+
+
+    def delete_resource(self, rid):
+        self.url_adr += rid
+        self.response = requests.delete(self.url_adr)
+        # self.json_data = json.loads(self.response.text)
+
+    def get_measurements(self):
+        measurements_list = []
+        measurement = None
+        page = None
+        self.response = requests.get(self.url_adr, params=self.payload)
+        self.json_data = json.loads(self.response.text)
+
+        # for item in self.json_data['resources']:
+        # print(self.json_data['page'])
+        for item in self.json_data['measurements']:
+            # measurement = Measurements()
+            # measurement.host = item['host']
+            # measurement.metric = item['metric']
+            # measurement.unit = item['unit']
+            # measurement.max_value = item['maxValue']
+            # measurement.complex_mes = item['complex']
+            # measurement.values = item['values']
+            # resources_list.append(resource)
+            # test line ****************************************
+            measurements_list.append((item['host'], item['metric'], item['unit'], item['maxValue'], item['complex'], item['values']))
+
+        print(self.json_data['page']['number'], self.json_data['page']['totalCount'], self.json_data['page']['size'])
+        # page.size = self.json_data['page']['number']
+        # page.number = self.json_data['page']['totalCount']
+        # page.total_count = self.json_data['page']['size']
+
+        return [measurements_list, page]
+
+    def get_measurements_guid(self, guid):
+        self.url_adr += guid
+        self.response = requests.get(self.url_adr)
+        self.json_data = json.loads(self.response.text)
+        # measurement = Measurements()
+        # measurement.host = item['host']
+        # measurement.metric = item['metric']
+        # measurement.unit = item['unit']
+        # measurement.max_value = item['maxValue']
+        # measurement.complex_mes = item['complex']
+        # measurement.values = item['values']
+        # return measurement
+
+    def get_measurements_guid_values(self, guid):
+        values = []
+        self.url_adr += (guid+'/')
+        self.response = requests.get(self.url_adr, params=self.payload)
+        self.json_data = json.loads(self.response.text)
+        return self.json_data
+        # return values
+
+    def delete_measurements_guid_values(self, guid):
+        self.url_adr += guid+"/values"
+        # print(self.url_adr)
+        self.response = requests.delete(self.url_adr)
+
+    def post_measurements(self):
+        #post with json
+        self.response = requests.post(self.url_adr, data=json.dumps(self.payload))
+
+
+    def delete_measurements_guid(self, guid):
+        self.url_adr += guid+"/"
+        self.response = requests.delete(self.url_adr)
+
+# testowanie
+# def main():
```

Comment:
Please, use django unit test module instead of testing it in main. Advantages:

- they run in closed environment so you don't have to mix testing code with production code,
- no more commenting code, tests simply are there and you can run them any time
- you can run them after any code change to check if everything still works as intended